samples/samples.rst

```diff
@@ -20,6 +20,7 @@ Samples and Demos
    application_development/*
    display/*
    shields/*
+   portability/*
```

Comment:
For consistency with the other sample sections (such as display and shields), you'll also need to create a file ``portability/portability.rst`` with this content (resolving the doc build errors you're seeing from shippable):
```
.. _portability-samples:

Portability Samples
###################

.. toctree::
   :maxdepth: 1
   :glob:

   **/*

```

Reply:
@dbkinder: 👍 thanks.